lib/pages/trending/widgets/info_card.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:soundspace/constants/style.dart';
+import 'package:soundspace/constants/music_list.dart';
+import 'package:soundspace/pages/trending/widgets/components/custom_list_tile.dart';
+
+class InfoCard extends StatelessWidget {
```

Comment:
This should probably be changed to a stateful widget so that it will be intractable in the future if I understand you design correctly.